recipe/build-cpp.sh

```diff
@@ -1,53 +1,70 @@
 #!/bin/bash
-
 set -ex
 
 export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CXX_STANDARD=17"
 
 if [[ "$CONDA_BUILD_CROSS_COMPILATION" == 1 ]]; then
-  (
     mkdir -p build-host
     pushd build-host
 
+    # save default compilation setup
+    export SAVED_CC="${CC}"
+    export SAVED_CXX="${CXX}"
+    export SAVED_CFLAGS="${CFLAGS}"
+    export SAVED_CXXFLAGS="${CXXFLAGS}"
+    export SAVED_LDFLAGS="${LDFLAGS}"
+    export SAVED_CMAKE_ARGS="${CMAKE_ARGS}"
+    export SAVED_PKG_CONFIG_PATH="${PKG_CONFIG_PATH}"
```

Comment:
This is not necessary, we open a new scope in line 8 (you removed that)

Reply:
Ok, makes sense, I'll fix it. Though this kind of thing is really too subtle to not be documented.